pywbem/_subscription_manager.py

```diff
@@ -385,21 +432,22 @@ def add_filter(self, server_id, source_namespace, query,
           query (:term:`string`):
             Filter query in the specified query language.
 
-          owned (:class: `pybool`) Defines whether a filter whose lifecycle
-            is owned by this server or not is to be created.
-            If True, a filter is
-            created that is defined by the life cycle of this server instance
-            within this subscription manager.  If False a  filter is created
-            that it may outlive the lifecycle of the server instance
-            and must be removed manually by the user (for example using
-            the `remove_filter` function.
 
           query_language (:term:`string`):
             Query language for the specified filter query.
 
             Examples: 'WQL', 'DMTF:CQL'.
 
-        filter_id(:term: `string`)
+          owned (:class:`py:bool`)
+            Defines whether a filter whose lifecycle is owned by this server
+            or not is to be created. If True, a filter is
+            created that is defined by the life cycle of this server instance
+            within this subscription manager.  If False a  filter is created
+            that it may outlive the lifecycle of the server instance
+            and must be removed manually by the user (for example using
+            the `remove_filter` function.
```

Comment:
Incorrect markup for reference to `remove_filter()`. Change to:

    :meth:`~pywbem.WBEMSubscriptionManager.remove_filter`